app/core/management/commands/mysuperuser.py

```diff
@@ -9,7 +9,7 @@
 
 
 class Command(BaseCommand):
-    def handle(self, *args, **options):
+    def handle(self):
```

Comment:
I'm not sure this change is okay! Doesn't django expect to pass these arguments?

Reply:
And will this kind of change be made automatically?

Reply:
Ooops no I broke that fixing it now